apps/prairielearn/src/pages/courseSyncs/courseSyncs.js

```diff
@@ -24,84 +25,63 @@ router.get(
       throw new HttpStatusError(403, 'Access denied (must be course editor)');
     }
 
-    const jobSequencesResult = await sqldb.queryAsync(sql.select_sync_job_sequences, {
-      course_id: res.locals.course.id,
-    });
-    res.locals.job_sequences = jobSequencesResult.rows;
+    const jobSequences = await queryRows(
+      sql.select_sync_job_sequences,
+      { course_id: res.locals.course.id },
+      JobSequenceRowSchema,
+    );
 
-    const result = await sqldb.queryAsync(sql.question_images, {
-      course_id: res.locals.course.id,
-    });
-    res.locals.images = result.rows;
-    res.locals.imageSyncNeeded = false;
+    const images = await queryRows(
+      sql.question_images,
+      { course_id: res.locals.course.id },
+      ImageRowSchema,
+    );
 
     if (config.cacheImageRegistry) {
       const ecr = new ECR(makeAwsClientConfig());
-      await async.each(res.locals.images, async (image) => {
-        var repository = new DockerName(image.image);
+      await async.each(images, async (image) => {
```

Comment:
This is using `async.each` instead of `async.eachSeries`, meaning this is done in parallel. I did not change that, but I just want to check that this is intentional, otherwise this could be changed to `async.eachLimit` or to a for loop.

Reply:
Done.